package.json

```diff
@@ -21,7 +21,9 @@
     "lint": "elm-format --validate packages && yarn eslint && yarn prettier && stylelint '**/*.scss'",
     "lint:fix": "elm-format --yes packages && yarn eslint --fix && yarn prettier --write",
     "clean:storybook": "rimraf 'storybook/public'",
-    "clean": "lerna run clean && yarn gatsby clean && yarn clean:storybook && rimraf 'elm-stuff'",
+    "clean:stories": "rimraf './packages/component-library/stories/*.d.ts' './packages/component-library/stories/*.js.map' './packages/component-library/stories/*.js'",
+    "clean:draft-stories": "rimraf './draft-packages/stories/*.d.ts' './draft-packages/stories/*.js.map' './draft-packages/stories/*.js'",
```

Comment:
You shouldn't need these two scripts unless you're running `yarn lerna run publish` locally (which a large majority of people will never need). I've added it here for convenience.

Reply:
What should really happen is `yarn lerna run prepublish` shouldn't generate declaration files/source maps for assets that we don't actually publish 🙃 Will be tackled in a separate housekeeping PR.